src/libs/ReportUtils.js

```diff
@@ -2154,6 +2157,7 @@ function getParentReport(report) {
 
 export {
     getReportParticipantsTitle,
+    getPersonalDetailsForLogin,
```

Comment:
What is this for?

Reply:
I used in in tooltips where only the login was available to get the accountID. Is there a better alternative?